hostapd/files/hostapd.conf.jinja

```diff
@@ -26,7 +25,7 @@ wmm_enabled={{ card_data.get('wmm_enabled') }}
 {%- endif %}
 {#- don't break compatibility with old config format #}
 {%- set ap_list_orig = card_data.get('ap_list', {}) %}
-{%- set ap_list = ap_list_orig.iteritems() if ap_list_orig is mapping else ap_list_orig %}
+{%- set ap_list = ap_list_orig.items() if ap_list_orig is mapping else ap_list_orig %}
```

Comment:
The `dictsort` Jinja filter is generally preferred over `items()` method, because it returns a copy of a dict, not view.
This is not the case here, but when the original dictionary would be somehow modified later, there could be troubles.

Reply:
I'd leave it as it is. Input data of a template is rarely modified and I doubt this to be ever the case for this formula.